### ISSUE TITLE
[DUPE of GH-13572 and GH-14911] Aria-label support in category facet buttons on search page 

### ISSUE THREAD

**Issue by Parthlakhani** [SAP/spartacus#12898]
Add aria-label in category facet buttons. 
Sub-categories are not accessible via keyboard. Add support for tabbing in sub-categories.

Related to #11204 

<img width="306" alt="Screen Shot 2021-06-21 at 3 49 58 PM" src="https://user-images.githubusercontent.com/72991814/122819358-5eb21580-d2a8-11eb-8440-f56d5761306e.png">

**Comment by saad-mo** [comment]
this will probably require breaking changes to the markup. 

@developpeurweb any suggestions for a solution?

**Comment by developpeurweb** [comment]
@saad-mo The issues on this ticket might get fixed when GH-11637 is done. I'll flag it as a potential dupe.

**Comment by developpeurweb** [comment]
Dupe of GH-13572 and GH-14911 to be closed when both are merged.

**Comment by Matejk00** [comment]
Closing because of duplicate.

**Comment by KateChuen** [comment]
Fixed and tested. Can access sub categories facet group using keyboard, plus there's a vocalized feedback.
